BridgeAppSDK/Localization.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Localization.swift
+//  BridgeAppSDK
+//
+//  Created by Shannon Young on 2/17/16.
+//  Copyright © 2016 Sage Bionetworks. All rights reserved.
+//
+
+import UIKit
+
+class Localization: NSObject {
+    
+    static var localeBundle = {
+        return NSBundle.init(forClass: Localization.classForCoder())
+    }()
+    
+    static func localizedString(key: String) -> String {
+        return NSLocalizedString(key, tableName: nil, bundle: localeBundle, value: key, comment: "")
+    }
+    
+    // List of the strings used in this module. This func is never called but is included to allow running 
```

Comment:
Never mind, I was confused about what the argument to localizedString would be. I think this will work and we'll see right away in the UI if a string hasn't been added to this list. Cool!